Manual.md

```diff
@@ -781,6 +781,18 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gemspec` For packages that are built into gems using .gemspec files
+and then packaged. This build style is meant for packages that have
+C extensions and as such require patching to support cross compilation.
+
+- `gem` For packages that are installed using gems from [RubyGems](https://rubygems.org/)
+or other providers. Packages that use this build style must follow the ruby-`pkgname`
+naming convention with `pkgname` being the name of the package according
+build style. The `gem_cmd` variable can be used to set the name of the gem binary.
+
+Packages under this build style are always `noarch`, if your gem provides compiled
+extensions use the `gemspec` build style that can handle that.
```

Comment:
_C_ extensions?

Reply:
`that can handle that.`

Somehow that sounds pretty "meh" to me. Maybe just say "use the `gemspec` build style instead".